raiden/transfer/node.py

```diff
@@ -1309,7 +1316,8 @@ def update_queues(iteration: TransitionResult[ChainState], state_change: StateCh
             queue.append(event)
 
         if isinstance(event, ContractSendEvent):
-            chain_state.pending_transactions.append(event)
+            if is_transaction_pending(chain_state, event, state_change):
+                chain_state.pending_transactions.append(event)
```

Comment:
this doesn't seem specific to "send close".

could you please explain why this wasn't affecting other types of transactions?

Reply:
Other transactions don't throw exceptions that pull down the node so that's why you don't see any error but wrong anyway

Reply:
thanks